js/src/wasm/WasmFrame.h

```diff
@@ -317,7 +319,7 @@ class DebugFrame {
   }
   static constexpr uint32_t hasSpilledRegisterRefResultBitMask(size_t n) {
     MOZ_ASSERT(n < MaxRegisterResults);
-    union Flags flags = {.allFlags = 0};
+    union Flags flags{0};
```

Comment:
Alternatively I believe this could just change `{0}` to `{}` and rely on value-initialization of the struct?